releasenotes/notes/apm-symdb-endpoint-ca0c373b539df46d.yaml

```
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    APM: Added a new endpoint in the trace-agent api `/symdb/v1/input` that acts as a reverse proxy forwarding requests to a specific event platform intake. The feature using this is currently in development.
```

```suggestion
    APM: Added a new endpoint in the trace-agent api `/symdb/v1/input` that acts as a reverse proxy forwarding requests to Datadog. The feature using this is currently in development.
```
Customers don't necessarily understand what event platform intake means.